src/main/java/org/chtracker/web/PreventiveTreatmentTypeController.java

```diff
@@ -12,25 +10,17 @@
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
-@RequestMapping("/TreatmentType")
-public class TreatmentTypeController {
+@RequestMapping("/preventive-treatment-types")
+public class PreventiveTreatmentTypeController {
 
-	private final AbortiveTreatmentTypeRepository abortiveTreatmentTypeRepository;
 	private final PreventiveTreatmentTypeRepository preventiveTreatmentTypeRepository;
 
 	@Autowired
```

Comment:
@Autowired in here is not neccessary. Presence of constructor with Field PreventiveTreatmentTypeRepository already injects it.

Reply:
Thanks for the info! Funny, I didn't know it. They added it in Spring 4.3. though I'd rather stick to verbose approach in order not to remember yet another "default" rule.